src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -656,6 +656,23 @@ export type PullRequest = {
   externalUrl: string;
 };
 
+type IntegrationDialog = {
+  actionText: string;
+  body: string;
+};
+
+type IntegrationAspects = {
+  alerts?: Array<{type: 'info' | 'warning'; text: string; icon?: string}>;
```

Comment:
Tiny nit, how about `React.ComponentPros<typeof Alert>`

Reply:
@EvanPurkhiser won't work because it doesn't have the `text` attribute

Reply:
Can we just union it in?